third_party/mkdocs-material/overrides/partials/header.html

```diff
@@ -0,0 +1,170 @@
+<!--
+  Copyright (c) 2016-2023 Martin Donath <martin.donath@squidfunk.com>
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to
+  deal in the Software without restriction, including without limitation the
+  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+  sell copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+<!-- Determine base classes -->
+{% set class = "md-header" %}
+{% if "navigation.tabs.sticky" in features %}
+  {% set class = class ~ " md-header--lifted" %}
+{% endif %}
+
+<!-- Header -->
+<header class="{{ class }}" data-md-component="header">
+  <nav
+    class="md-header__inner md-grid"
+    aria-label="{{ lang.t('header') }}"
+  >
+
+    <!-- Link to community home -->
+    <a
+      href="{{ config.extra.community_homepage | d(nav.homepage.url, true) | url }}"
+      title="OpenXLA"
+      class="md-header__button md-logo"
+      aria-label="OpenXLA"
+      data-md-component="logo"
+    >
+      {% include "partials/logo.html" %}
+    </a>
+
+    <!-- Link to IREE home -->
+    <a
+      href="{{ config.extra.homepage | d(nav.homepage.url, true) | url }}"
+      title="IREE"
+      class="md-header__button md-logo"
+      aria-label="IREE"
+      data-md-component="logo"
+    >
+      <div class="md-header__title">
+          <div class="md-nav__title">
+            {{ config.site_name }}
+          </div>
+      </div>
```

Comment:
Need to set some class on this that respects the color scheme
![image](https://user-images.githubusercontent.com/4010439/219502991-ffff659e-6ea0-4cef-a337-ba2d707dcdb1.png)
![image](https://user-images.githubusercontent.com/4010439/219503014-e4d798c3-bd5e-45b5-b116-ce95b244c961.png)

(also why is the nav indented...? might need to tweak the `<div>` I added there...)